### PR TITLE
[10.0][FIX]hr_employee_product

### PR DESCRIPTION
**Pull request by AaronHForgeFlow**
@bjeficent 

If product is employee, the employee gets error when doing actions in the timesheet

### CODE REVIEW

**Review by bjeficent — APPROVED** [review]
LGTM

**Review by nikul-serpentcs — APPROVED** [review]
Code Review LGTM :+1:

**Comment by nikul-serpentcs** [comment]
@pedrobaeza Fast trecking

File: hr_employee_product/security/product_security.xml [hunk 1]
```diff
@@ -14,5 +14,15 @@
             <field name="domain_force">[(1, '=', 1)]</field>
             <field name="groups" eval="[(4,ref('hr.group_hr_manager'))]"/>
         </record>
+
+        <record id="property_rule_product_employee_hr_manager" model="ir.rule">
+            <field name="name">HR Managers can view products that represent employees</field>
+            <field name="model_id" ref="product.model_product_template"/>
+            <field name="groups" eval="[(4,ref('base.group_user'))]"/>
```
Comment (pedrobaeza) [inline hr_employee_product/security/product_security.xml:21]: But this is not HR managers...

Comment (nikul-serpentcs) [reply hr_employee_product/security/product_security.xml]: It's mean adding HR managers Groups  'group_hr_manager' also

Comment (pedrobaeza) [reply hr_employee_product/security/product_security.xml:21]: I still don't understand it...

**Review by pedrobaeza — CHANGES_REQUESTED** [review]
My comment is not attended

**Comment by lmignon** [comment]
@aheficent can you check the comments from @pedrobaeza

**Comment by AaronHForgeFlow** [comment]
The thing is, we can use these products in the timesheets in order to know what GL account will use depending on the employee. If we don't include this PR then we will not be able to do that because normal uses will get errors when filling their timesheets.

Maybe giving read access is not the solution, perhaps it is better to use root user when reading the product in the other module. I am not sure.

**Comment by pedrobaeza** [comment]
The problem is that the code don't honor the previous title: `HR Managers can view products that represent employees`. Any way, giving read access for all employees to product templates is not the best option. If you have an access error, you have to:

* If there's a compute field that use the product, put `compute_sudo=True`.
* If the field is shown in certain view, you have to add the minimum group with acces to that view (not to the field).

**Comment by AaronHForgeFlow** [comment]
Closing, this is not correct.

**Pull request closed without merging**